ion/writer.go

```diff
@@ -78,6 +78,12 @@ type Writer interface {
 	// Annotations adds multiple annotations to the next value written.
 	Annotations(values ...string) error
```

Comment:
If we're leaving the `string`-centric `Annotations` method here: are `string`s that aren't in the symbol table quietly added to it before the value is written out, or will it produce an `error`? Either way, that behavior should be clarified in the documentation.